readme.md

```diff
@@ -232,6 +233,7 @@ Made with Electron.
 - [electron-input-menu](https://github.com/parro-it/electron-input-menu) - Context menu for input elements.
 - [chrome-tabs](https://github.com/adamschwartz/chrome-tabs) - Chrome like tabs.
 - [titlebar](https://github.com/kapetan/titlebar) - Emulate the OS X window titlebar.
+- [express-web-components](https://github.com/scramjs/express-web-components) - Declaratively build Express.js applications with web components.
```

Comment:
Out of scope for this list. The `Component` section is components you can use in the renderer to make apps, like widgets.